carbon/datapoint_test.go

```diff
@@ -0,0 +1,100 @@
+package carbon
+
+import "testing"
+
+func Test_NewDatapoint(t *testing.T) {
+	const (
```

Comment:
It would be cleaner to use table driven tests:
https://github.com/golang/go/wiki/TableDrivenTests

Reply:
Fixed